snapcraft.yaml

```diff
@@ -0,0 +1,41 @@
+name: subiquity
+version: "0.0.32"
+summary: Ubuntu installer
+description: The Ubuntu server installer
+confinement: classic
+
+apps:
+  subiquity:
+    command: usr/bin/subiquity
+  console-conf:
```

Comment:
console-conf comes from the same code as subiquity, we'll eventually need it some way or another as a snap, so it will already be done and available.